test/transaction.test.ts

```diff
@@ -291,9 +291,7 @@ describe("Path Payment", () => {
         sendAmount: "5",
       })
       .build();
-    sourceKp.sign(txn);
```

Comment:
unrelated fix. This test this fails intermintedly when usdc-xlm order book is volatile on testnet, so fixing to just check  the operation type (.swap is still submitting to the network and uses .pathPay)